hugr-core/src/extension/op_def.rs

```diff
@@ -217,11 +268,12 @@ impl From<CustomValidator> for SignatureFunc {
 }
 
 impl SignatureFunc {
-    fn static_params(&self) -> &[TypeParam] {
-        match self {
+    fn static_params(&self) -> Result<&[TypeParam], SignatureError> {
+        Ok(match self {
             SignatureFunc::PolyFuncType(ts) => ts.poly_func.params(),
             SignatureFunc::CustomFunc(func) => func.static_params(),
-        }
+            SignatureFunc::MissingComputeFunc => return Err(SignatureError::MissingComputeFunc),
```

Comment:
if `SignatureFunc::ComputeFunc` specified which kind of function was missing, you could handle the "missing validation" by returning empty